spot_ros2_control/examples/src/joint_command_passthrough.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2024 Boston Dynamics AI Institute LLC. All rights reserved.
+
+// Note(llee): I think I would like to move this to `action_stack_spot` because so far it's pretty specific to Spot
+
+#include <chrono>
+#include <functional>
+#include <map>
+#include <memory>
+#include <ranges>
+#include <string>
+#include <vector>
+
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/joint_state.hpp"
+#include "spot_ros2_control/spot_joint_map.hpp"
+#include "std_msgs/msg/float64_multi_array.hpp"
+
+class JointCommandPassthrough : public rclcpp::Node {
+ public:
+  JointCommandPassthrough() : Node("joint_passthrough") {
```

Comment:
Can you add the robot name as a parameter? `Ethernet0` isn't a universal thing both internally (you can have `Ethernet1`, etc) and externally where you might not have a name at all.

Reply:
@amessing-bdai yes, I really really really want to make it a parameter but would you be hugely opposed to that as going up as a PR tomorrow?

Reply:
No, so long as it is in the near future.

Reply:
Most definitely this week, but this is blocking a bdai pr I want in just to get functionality all committed